### PR TITLE
Reports API moved from stable to preview version with addition of documentation of a couple of new APIs

### PR DESCRIPTION
**Pull request by shalinved**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by shalinved** [comment]
@ravbhatnagar, @ms-premp

**Review by sergey-shandar — CHANGES_REQUESTED** [review]
Why are you moving from `stable` to `preview`? Usually, we don't allow this.

**Comment by shalinved** [comment]
@sergey-shandar - We had a couple of meeting with @ravbhatnagar on moving the APIs from stable to preview. These APIs were never meant to be released as a stable version. They were meant to unblock internal Microsoft teams integrating with us. We didn't create any SDKs, didn't publish the APIs and have already verified that all the requests we received are from internal Microsoft teams.

We understand that once the APIs are available publicly, someone could build SDKs on top of these APIs and that experience might get broken but the data we have combined with absence of any API announcements suggests there is a very slim chance of that happening.

**Review by sergey-shandar — APPROVED** [review]
We still need a formal approval from ARM

File: specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-09-01-preview/costmanagement.json
Comment (shalinved) [reply specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-09-01-preview/costmanagement.json]: Done

File: specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-09-01-preview/costmanagement.json
Comment (shalinved) [reply specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-09-01-preview/costmanagement.json]: Updated to POST

File: specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-09-01-preview/costmanagement.json
Comment (shalinved) [reply specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-09-01-preview/costmanagement.json]: Updated to POST

**Review by sergey-shandar — APPROVED** [review]

**Comment by ravbhatnagar** [comment]
Signing off from ARM side.

**Comment by sergey-shandar** [comment]
@shalinved please, resolve conflicts

**Pull request merged**
